### PR TITLE
yes: add example

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by tricantivu — COMMENTED** [review]
What do you mean by "default option"?

**Comment by Managor** [comment]
Occasionally programs prompt you for `[Y/n]` on `[y/N]`. This allows you to always select the one that the program has as the "default". Only using `yes` would constantly output `y` which means that [y/N] would be overridden.

**Comment by tricantivu** [comment]
> Occasionally programs prompt you for `[Y/n]` or `[y/N]`. `yes ''` allows you to always select the one that the program has as the "default". Only using `yes` would constantly output `y` which means that [y/N] would be overridden.

I see, maybe `archinstall` and `pylint-config` are examples of such programs, right?

File: pages/common/yes.md [hunk 1]
```diff
@@ -15,3 +15,7 @@
 - Accept everything prompted by the `apt-get` command:
 
 `yes | sudo apt-get install {{program}}`
+
+- Repeatedly output newline to always accept the default option:
```
Comment (tricantivu) [inline pages/common/yes.md:19]:
```suggestion
- Repeatedly output a newline to always accept the default option of a prompt:
```

**Review by tricantivu — COMMENTED** [review]
The example seems useful, however, I have a minor suggestion.

**Comment by Managor** [comment]
> I see, maybe `archinstall` and `pylint-config` are examples of such programs, right?

For me it was `yay`. It prompts to delete build deps after install with [y/N]

**Review by fazlearefin — APPROVED** [review]

**Comment by fazlearefin** [comment]
Thanks for your contribution @Managor . This is an useful example.

**Review by kbdharun — APPROVED** [review]
LGTM, thanks for the addition.

**Pull request merged**